### PR TITLE
Mention that new files should get the SPDX header

### PR DESCRIPTION
**Pull request by cmarqu**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -72,6 +72,10 @@ All changes which should go into the main codebase of cocotb must follow this se
      Use this text to discuss things which are not obvious from the code, especially *why* changes were made.
      Include the GitHub issue number (if one exists) in the form "Fixes #nnn" ([read more about that](https://help.github.com/articles/closing-issues-using-keywords/)).
      Keep each description line below 72 characters.
+- Use the following header for new files:
+  > \# Copyright cocotb contributors
+  > \# Licensed under the Revised BSD License, see LICENSE for details.
+  > \# SPDX-License-Identifier: BSD-3-Clause
```
Comment (eric-wieser) [inline CONTRIBUTING.md:78]: This doesn't render well. Can you use a code block instead?

Comment (cmarqu) [reply CONTRIBUTING.md]: Oops, fixed. It rendered fine locally...

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -72,7 +72,12 @@ All changes which should go into the main codebase of cocotb must follow this se
      Use this text to discuss things which are not obvious from the code, especially *why* changes were made.
      Include the GitHub issue number (if one exists) in the form "Fixes #nnn" ([read more about that](https://help.github.com/articles/closing-issues-using-keywords/)).
      Keep each description line below 72 characters.
-
+- Use the following header for new files:
+```
```
Comment (eric-wieser) [inline CONTRIBUTING.md:76]: This should be indented two spaces to match the list item (and the lines below)

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -73,11 +73,11 @@ All changes which should go into the main codebase of cocotb must follow this se
      Include the GitHub issue number (if one exists) in the form "Fixes #nnn" ([read more about that](https://help.github.com/articles/closing-issues-using-keywords/)).
      Keep each description line below 72 characters.
 - Use the following header for new files:
-```
-# Copyright cocotb contributors
-# Licensed under the Revised BSD License, see LICENSE for details.
-# SPDX-License-Identifier: BSD-3-Clause
-```
+  ```
```
Comment (eric-wieser) [inline CONTRIBUTING.md:76]:
```suggestion
  ```python
```

**Pull request merged**